build.gradle

```diff
@@ -13,13 +16,14 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'io.cucumber', name: 'cucumber-java', version: '6.9.1'
-    implementation 'com.github.javafaker:javafaker:1.0.2'
-    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.7.0'
-    compile group: 'io.appium', name: 'java-client', version: '7.4.1'
-    compile 'io.github.cdimascio:dotenv-java:2.2.0'
-    testCompile group: 'io.cucumber', name: 'cucumber-junit', version: '6.9.1'
-
+    implementation('com.github.javafaker:javafaker:1.0.2')
+    implementation('io.github.cdimascio:dotenv-java:2.2.0')
+    implementation(group: 'io.cucumber', name: 'cucumber-java', version: '6.9.1')
+    implementation group: 'junit', name: 'junit', version: '4.13.1'
```

Comment:
Why are you downgrading this version?
Is it because of Allure?

Reply:
In the Readme file, I explain this: "It is recommended to work with JUnit4, because the @RunWith annotation of the test runners is typical of JUnit 4, for use with JUnit 5 and allure it is required to use the engine-vintage, but when doing so JUnit is not able to recognize the tests de cucumber, in addition to generating some unexpected behaviors and errors, which makes the project unstable."